draft-deshpande-rats-multi-verifier.md

```diff
@@ -65,14 +65,33 @@ IETF RATS Architecture, defines the key role of a Verifier.  In a complex system
+In a Composite Attester system, it may not be possible for a single Verifier to possess all the capabilities or information required to conduct a complete appraisal of the Attester. Please refer to {{sec-need-multiverifier}} for motivation of this document. Multiple Verifiers need to collaborate to reach a conclusion on the appraisal and produce the Attestation Results.
 
 
 This document describes various topological patterns of multiple Verifiers that work in a coordinated manner to conduct appraisal of a Composite Attester to produce an Attestation Results.
 
+# Need for Multiple Verifiers
+{: #sec-need-multiverifier }
+To conduct the task of Evidence appraisal, a Verifier requires:
+1. Reference Values from trusted supply chain actors producing, aggregating, or administering Attesters (Reference Value Providers)
+2. Endorsements from trusted supply chain actors producing, certifying, or compliance checking Attesters (Endorsers)
+3. Appraisal Policy for Evidence, which is under the control of the Verifier Owner
+
+The Verifier inputs listed above are linked to the shape and form of the Attesters, i.e., the type of claims and the policies of appraisal of Evidence.
+Typically, Composite Attesters come with a varying degree of heterogeneity of Evidence formats, depending on the type of Attesting Environments that come with each Component Attester, for example, CPU variants or GPU/FPGA variants.
+An Attester's composition can change over time based on market dynamics and availability (e.g., a set of racks in a
+data center gets thousands of new FPGAs). It is highly unlikely that there is always one appropriate Verifier that satisfies all the requirements that a complex and changing composite Attesters impose. It may not be economically viable to build and maintain such degree of complexity, in a single Verifier, hence there is a need for a Multi Verifier System.
+
+In addition to above, there are several other factors that can lead to multi Verifier system, few are listed below:
+1. A part of a single component Verification policy may be internal to a Verifier Owner and it may not choose to reveal it to a monolithic Verifier.
+2. An Endorser may not wish to reveal its Reference Values or their lifecycle to a monolithic Verifier, hence may choose to keep its verification seperate.
+3. There may not be a single actor in the ecosystem, that can stand up and take ownership of Verifying every component Attester due to lack of knowledge, complexity, regulations or associated cost.
+4. Verifier services provided by component manufactures as well as Verifiers provided by the integrators, as well as Verifiers under local authority (close to the Attester) are the mix today and rarely it is just one of them.
```

Comment:
Lines 88-92 *may* be ok in terms of content (although I am not entirely sure about 2), but they need some more work on the exposition to improve clarity.

Besides, I don’t understand why they are grouped separately.  They seem like core arguments to me, i.e., not hierarchically inferior to the “market dynamics” argument above.

In general, I believe it’d be better to restructure the section in a cleaner fashion, e.g., with a bullet list where each separate point is clearly and concisely articulated.